source/backends/debian/debpkgindex.d

```diff
@@ -63,65 +63,115 @@ public:
         indexChanged = null;
     }
 
-    private void loadPackageLongDescs (DebPackage[string] pkgs, string suite, string section)
+    private immutable(string[]) findTranslations (const string suite, const string section)
     {
-        immutable enDescPath = buildPath ("dists", suite, section, "i18n", "Translation-en.%s");
-        string enDescFname;
+        import std.regex : ctRegex, matchFirst;
+
+        immutable inRelease = buildPath (rootDir, "dists", suite, "InRelease");
+        auto regex = ctRegex!(r"Translation-(\w+)$");
+        auto ret = appender!(string[]);
 
         try {
-            enDescFname = downloadIfNecessary (rootDir, tmpDir, enDescPath);
-        } catch (Exception e) {
-            logDebug ("No long descriptions for %s/%s: %s", suite, section, e.msg);
-            return;
+            const inReleaseContents = getFile (inRelease);
+
+            foreach (entry; inReleaseContents) {
+                auto match = entry.matchFirst(regex);
+
+                if (match.empty)
+                    continue;
+
+                if (!ret.data.canFind (match[1]))
```

Comment:
Jup, I know and it's annoying - but you can use an associative array instead of a set.
See https://github.com/ximion/appstream-generator/blob/master/source/engine.d#L567 for an example.
(actually, we could make our own set type with this)